codebases/omgevingsbeleid.html

```diff
@@ -0,0 +1,109 @@
+---
+layout: codebase
+title: Codebases we work with - Digital Omgevingsbeleid
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+  </div>
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-maturity-detail-header">
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <h1>Digital Omgevingsbeleid</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>The Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. The Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
```

Comment:
```suggestion
      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
```

Reply:
@Ainali, why did you reject the date move?

Reply:
I didn't reject it, I committed all your suggestions as they came up, not sure what GitHub is up to.

Reply:
Bizarre.

Reply:
Changed it manually now